app/views/beaches/show.html.erb

```diff
@@ -33,8 +33,15 @@
 </p>
 
 <% @reviews.each do |review| %>
-  <%= review.rating %>
-  <%= review.description %>
+  <li><%= review.rating %></li>
+  <% unless review.description.nil? %>
+    <li><%= review.description %></li>
+  <% end %>
+  <li><%= link_to "Delete Review", "/beaches/#{@beach.id}/reviews/#{review.id}",
```

Comment:
There's no helper path for this. We tried to make destroy a shallow route, and then use the helper path review_path(review), but it didn't work.